src/Compilers/VisualBasic/Portable/Compilation/MethodCompiler.vb

```diff
@@ -257,7 +257,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 If (compiler.GlobalHasErrors OrElse moduleBeingBuiltOpt.SourceModule.HasBadAttributes) AndAlso Not hasDeclarationErrors AndAlso Not diagnostics.HasAnyErrors Then
                     ' If there were errors but no diagnostics, explicitly add
                     ' a "Failed to emit module" error to prevent emitting.
-                    diagnostics.Add(ERRID.ERR_ModuleEmitFailure, NoLocation.Singleton, moduleBeingBuiltOpt.SourceModule.Name)
+                    diagnostics.Add(ERRID.ERR_ModuleEmitFailure, NoLocation.Singleton, moduleBeingBuiltOpt.SourceModule.Name,
+                        If(compiler.GlobalHasErrors, CodeAnalysisResources.UnableToDetermineSpecificCauseOfFailure, CodeAnalysisResources.ModuleHasInvalidAttributes))
```

Comment:
>CodeAnalysisResources.UnableToDetermineSpecificCauseOfFailure, CodeAnalysisResources.ModuleHasInvalidAttributes [](start = 53, length = 111)

This is likely to cause localization problems, I think we are using LocalizableResourceString in situations like this.